code/modules/mob/living/carbon/human/species.dm

```diff
@@ -294,12 +316,33 @@
 					S = frills_list[H.dna.features["frills"]]
 				if("horns")
 					S = horns_list[H.dna.features["horns"]]
+				if("ears")
+					S = ears_list[H.dna.features["ears"]]
 				if("body_markings")
 					S = body_markings_list[H.dna.features["body_markings"]]
 
 			if(!S || S.icon_state == "none")
 				continue
+
+			//A little rename so we don't have to use tail_lizard or tail_human when naming the sprites.
+			if(bodypart == "tail_lizard" || bodypart == "tail_human")
+				bodypart = "tail"
+			else if(bodypart == "waggingtail_lizard" || bodypart == "waggingtail_human")
+				bodypart = "waggingtail"
```

Comment:
The idea of tail_lizard and tail_human isn't to distinguish between two species, but between two different tail lists. If another species wants to use lizard or human tails, they wouldn't be able to if I did this.

It's not super elegant but the alternative was defining exactly which individual accessories every species can pick and I figured that there are enough differences between lizard and mammal tails to warrant two lists.